### DISCUSSION TITLE
Updated Album description look

### DISCUSSION THREAD

**Discussion by KillerParticles** [ciderapp/Cider#771] [Added Features]
The album description box we have now is very hard to read, unpleasant to look at and even cuts off some parts of the text. 
![image](https://user-images.githubusercontent.com/93286201/161088227-291382d6-c0b2-44eb-9269-db7f8910fa53.png)

Instead, I propose its updated to look more modern and alike to the Apple Music aesthetic, I have included a concept image below that I made to reflex the changes I want.
![AlbumDescriptionConcept](https://user-images.githubusercontent.com/93286201/161088652-b9c902a6-ba08-4818-853f-7e22618d32c9.png)

**Comment by Amaru8** [comment]
Locking discussion, because the requested feature has been added to Cider.
